src/itemdrop.zig

```diff
@@ -414,8 +414,8 @@ pub const ItemDropManager = struct { // MARK: ItemDropManager
 				continue;
 			}
 			const hitbox = main.game.Player.outerBoundingBox;
-			const min = user.player.pos + hitbox.min;
-			const max = user.player.pos + hitbox.max;
+			const min = user.player().pos + hitbox.min;
```

Comment:
There are no stable pointers, so between loading the pointer and accessing .pos, the SparseSet which uses a List underneath could get resized causing a segfault here.

Please figure out a reliable solution for accessing entity values.